### DISCUSSION TITLE
list of which clients support intro skip buttons

### DISCUSSION THREAD

**Discussion by jumoog** [intro-skipper/intro-skipper#409] [General]
Let us know if your client supports the official and plugin buttons.

| Client              | offical button | plugin button |
|---------------------|---------|---------------|
| Jellyin Web         | x       | x             |
| Jellyfin Android    | x       | x             |
| Jellyfin Android TV | x       |               |
| webOS (LG)          | x       | x             |

**Comment by AbandonedCart** [comment]
| Client | offical button | plugin button | EDL |
|----|---|---|---|
| Jellyin Web | x | x |   |
| Jellyfin Android | x  | x |   |
| Jellyfin Android TV | x |   |   |
| webOS (LG) | x | x |   |
| Kodi |   |   | x  |

Do we want to include EDL?
